test/test_transforms_v2_refactored.py

```diff
@@ -5046,3 +5046,67 @@ def test_transform_error_cuda(self):
                 ValueError, match="Input tensor should be on the same device as transformation matrix and mean vector"
             ):
                 transform(input)
+
+
+def make_image_numpy(*args, **kwargs):
```

Comment:
`make_image_pil` is only ever used in this file (`test/test_transforms_v2_consistency.py`), yet it lives in `common_utils.py`. Maybe we should align where we put these `make_input` stuff?